### ISSUE TITLE
*Started* is not a json file issue

### ISSUE THREAD

**Issue by dissmaterial** [bellingcat/tiktok-hashtag-analysis#14]
I have been having an issue after installing tiktok-scraper

if I enter: 
python run_downloader.py -t london paris newyork -p

output:
Hashtags to scrape: ['london', 'paris', 'newyork']
Something's wrong with what is returned by tiktok-scraper for the hashtag london - *Started* is not a json file.

tiktok-scraper returned - TikTok Scraper Started

Something's wrong with what is returned by tiktok-scraper for the hashtag paris - *Started* is not a json file.

tiktok-scraper returned - TikTok Scraper Started

Something's wrong with what is returned by tiktok-scraper for the hashtag newyork - *Started* is not a json file.

tiktok-scraper returned - TikTok Scraper Started

when I installed tiktok scraper - there were no issues with its install. I was wondering what the potential issue could be?
Please note that I am using a windows pc
Node v16.17.1 is installed
Python 3.10.7 is also installed

Thanks for the help!

for the sake of clarity - here was my output when I installed tiktok-scraper

npm i -g tiktok-scraper

output:
npm WARN deprecated request-promise@4.2.6: request-promise has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142

changed 270 packages, and audited 271 packages in 13s

15 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

**Comment by dissmaterial** [comment]
@trislee @binlabs

**Comment by trislee** [comment]
Yes, this appears to be a problem with the tiktok-scraper dependency. I'm currently looking into alternative scrapers.

**Comment by cutterkom** [comment]
I have the same problem

**Comment by msramalho** [comment]
I've added this note to the readme for the time being, the issue comes from a crucial dependency like trislee mentioned:

> IMPORTANT NOTE: this tool relies on [drawrowfly/tiktok-scraper](https://github.com/drawrowfly/tiktok-scraper) which seems to be broken at time of writing and without updates for some time with several open issues ([796](https://github.com/drawrowfly/tiktok-scraper/issues/796) [#799](https://github.com/drawrowfly/tiktok-scraper/issues/799)) that need to be fixed before this library can work smoothly :/

**Comment by trislee** [comment]
Switched to working scraper in commit https://github.com/bellingcat/tiktok-hashtag-analysis/commit/900d6adc69f1e59cc9c405dcbf4792f36f6960c8.
